src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,56 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    height: #{$nav-item-height};
+    margin-right: 20px;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    padding: (($nav-item-height - $nav-link-height) / 2) 0;
+
+    font-size: 12px;
+    line-height: #{$nav-link-height};
+    font-weight: 500;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: inherit;
+
+    transition: color 0.3s;
+
+    &:hover {
+      color: #00acdc;
```

Comment:
Checklist p11. You can put the color in variables, since you use the same color more than once